### PR TITLE
issue #31642 Remove Work Order Item characteristic usage

### PR DESCRIPTION
**Pull request by ChristopherCotnoir**

### CODE REVIEW

**Comment by ChristopherCotnoir** [comment]
Requires xtuple/qt-client#1839.

File: foundation-database/public/patches/populate_source.sql [hunk 1]
```diff
@@ -706,36 +707,19 @@ select createDoctype(41, --pDocAssNum
...
-                     'formatWonumber(wo_id)', --pName
-                     'formatWonumber(wo_id)', --pDesc
-                     '', --pWidget
-                     '', --pJoin
-                     'wo_id', --pParam
-                     'workOrder', --pUi
-                     '', -- pPriv, set this to 'MaintainWorkOrders' when Incident #28911 is implemented
-                     'Manufacture' --pModule
-);
 select createDoctype(42, --pDocAssNum
```
Comment (andersdd) [inline foundation-database/public/patches/populate_source.sql]: If you are going to delete a source, you will need a migration script for any customer data.

Comment (ChristopherCotnoir) [reply foundation-database/public/patches/populate_source.sql]: What data would need to be migrated?

Comment (andersdd) [reply foundation-database/public/patches/populate_source.sql]: Anything in the `charass` table with a charass_target_type of `WO`.

Comment (gilmoskowitz) [reply foundation-database/public/patches/populate_source.sql]: But what about #3130 and xtuple/qt-client#1744?

**Pull request merged**